scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
+// Meter
+// ==========================================================================
+meter {
+	-webkit-appearance: none;
+	width: 550px;
+	height: 25px;
+}
+
+.meter::-webkit-meter-bar, .multiColor::-webkit-meter-bar, .stripMeter::-webkit-meter-bar {
+	box-shadow: inset 0 0 7px -2px rgba(0,0,0,0.35);
+	background: #f1f1f1;
+}
+
+.meter::-webkit-meter-optimum-value {
+	background: #37bfc9;
+}
+
+
+.multiColor::-webkit-meter-optimum-value {
```

Comment:
Your naming scheme goes against the previously agreed upon naming conventions.  This project is going with a BEM naming convention, without camel case.